### PR TITLE
Refactoring: Project repo name is updated to vsphere-storage-for-docker

### PR DESCRIPTION
**Pull request by shuklanirdesh82**
With this PR our repo will be renamed to `vsphere-storage-for-docker` and all the references will be `docker-volume-vsphere` ---> `vsphere-storage-for-docker`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Code Contribution guidelines
 ### Dev Setup and debugging help
-Read the [FAQ on the Wiki](https://github.com/vmware/docker-volume-vsphere/wiki#faq)
+Read the [FAQ on the Wiki](https://github.com/vmware/vsphere-storage-for-docker/wiki#faq)
```
Comment (ashahi1) [inline CONTRIBUTING.md]: I know this is not part of PR but url shows name as "VMware Docker Volume Driver for vSphere"

File: Interop.md [hunk 2]
```diff
@@ -6,20 +6,20 @@ The vDVS plugin works well with all vSphere features and deviations if any are a
 
 SvMotion/XvMotion is not supported for VMs that use vDVS volumes.
 
-- SvMotion/XvMotion causes vDVS volumes to be migrated out of the datastore location and hence causing the plugin to lose access to the volume. [#1618](https://github.com/vmware/docker-volume-vsphere/issues/1618)
+- SvMotion/XvMotion causes vDVS volumes to be migrated out of the datastore location and hence causing the plugin to lose access to the volume. [#1618](https://github.com/vmware/vsphere-storage-for-docker/issues/1618)
 
 **Storage DRS**
 
-Storage DRS is not supported for VMs that use vDVS volumes. 
+Storage DRS is not supported for VMs that use vDVS volumes.
```
Comment (ashahi1) [inline Interop.md]: vDVS? Shall we change this too?

File: README.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+[![Go Report Card](https://goreportcard.com/badge/github.com/vmware/vsphere-storage-for-docker)](https://goreportcard.com/report/github.com/vmware/vsphere-storage-for-docker)
+[![Join the chat at https://gitter.im/vmware/vsphere-storage-for-docker](https://badges.gitter.im/vmware/vsphere-storage-for-docker.svg)](https://gitter.im/vmware/vsphere-storage-for-docker?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Docker Pulls](https://img.shields.io/badge/docker-pull-blue.svg)](https://store.docker.com/plugins/vsphere-docker-volume-service?tab=description) [![VIB_Download](https://api.bintray.com/packages/vmware/vDVS/VIB/images/download.svg)](https://bintray.com/vmware/vDVS/VIB/_latestVersion)
 
 
 # vSphere Docker Volume Service
 
 vSphere Docker Volume Service (vDVS) enables customers to address persistent storage requirements for Docker containers in vSphere environments. This service is integrated with [Docker Volume Plugin framework](https://docs.docker.com/engine/extend/). Docker users can now consume vSphere Storage (vSAN, VMFS, NFS) to stateful containers using Docker.
```
Comment (ashahi1) [inline README.md]: Same as above - vSphere Docker Volume Service

File: README.md [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
 
-[<img src="https://github.com/vmware/docker-volume-vsphere/blob/master/docs/misc/Docker%20Certified.png" width="180" align="right">](https://store.docker.com/plugins/vsphere-docker-volume-service?tab=description)vDVS is Docker Certified to use with Docker Enterprise Edition and available in [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description).
+[<img src="https://github.com/vmware/vsphere-storage-for-docker/blob/master/docs/misc/Docker%20Certified.png" width="180" align="right">](https://store.docker.com/plugins/vsphere-docker-volume-service?tab=description)vDVS is Docker Certified to use with Docker Enterprise Edition and available in [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description).
```
Comment (ashahi1) [inline README.md]: https://store.docker.com/plugins/vsphere-docker-volume-service?tab=description also needs to be updated later on. It still shows "vSphere Docker Volume Service"

File: README.md [hunk 5]
```diff
@@ -1,26 +1,26 @@
...
 
 To read more about code development and testing please read
-[CONTRIBUTING.md](https://github.com/vmware/docker-volume-vsphere/blob/master/CONTRIBUTING.md)
+[CONTRIBUTING.md](https://github.com/vmware/vsphere-storage-for-docker/blob/master/CONTRIBUTING.md)
 as well as the
-[FAQ on the project site](http://vmware.github.io/docker-volume-vsphere/documentation/faq.html).
+[FAQ on the project site](http://vmware.github.io/vsphere-storage-for-docker/documentation/faq.html).
```
Comment (ashahi1) [inline README.md]: Same as above http://vmware.github.io/vsphere-storage-for-docker/documentation/faq.html 
Later on we can update the webpage to shows vsphere-storage-for-docker

File: CONTRIBUTING.md (hunk 1 above)
Comment (shuklanirdesh82) [inline CONTRIBUTING.md]: Thanks for catching! no need for PR, I will update the title at github wiki page.

File: README.md (hunk 4 above)
Comment (shuklanirdesh82) [inline README.md]: good catch! I think @tusharnt is planning to followup with docker store team to update the title.

File: docs/index.md [hunk 6]
```diff
@@ -1,22 +1,22 @@
 ** vSphere Docker Volume Service - Introduction **
```
Comment (ashahi1) [inline docs/index.md]: May be later on we can change this one too - vSphere Docker Volume Service?

File: plugin_dockerbuild/Readme.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-### Support for building docker-volume-vsphere(DVS) as Docker Managed Plugin
+### Support for building vsphere-storage-for-docker(DVS) as Docker Managed Plugin
```
Comment (ashahi1) [inline plugin_dockerbuild/Readme.md]: "DVS" ?

**Pull request merged**